.github/workflows/smoke-test.yml

```diff
@@ -221,19 +252,20 @@ jobs:
           pytest --nbmake source/**/*.ipynb -n=auto
 
   r-test:
-    needs: rust-build-and-test
+    needs: rust-build
     runs-on: ubuntu-22.04
     env:
       OPENDP_LIB_DIR: ${{ github.workspace }}/libs
       LINTR_ERROR_ON_LINT: true
     steps:
+
       - name: Checkout repository
         uses: actions/checkout@v4
 
       - name: Download libs
         uses: actions/download-artifact@v4
         with:
-          name: libs
+          name: static-lib
```

Comment:
Add a note pointing out that R is different, and explaining why. Coming back in a few months it probably wont' be obvious why this distinction was needed.